armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -299,10 +306,12 @@ BITCOIN_VERSION="0.18.0"
 
 mkdir -p /usr/local/src/bitcoin
 cd /usr/local/src/bitcoin/
-curl --retry 5 -SL https://bitcoincore.org/keys/laanwj-releases.asc | gpg --import
+#curl --retry 5 -SL https://bitcoincore.org/keys/laanwj-releases.asc | gpg --import
```

Comment:
Nit: Could we remove commented-out lines?

Reply:
ACK